csi/src/mount.rs

```diff
@@ -303,7 +301,7 @@ pub fn blockdevice_mount(
 
 /// Unmount a block device.
 pub fn blockdevice_unmount(target: &str) -> Result<(), Error> {
-    let flags = UnmountFlags::FORCE;
+    let flags = UnmountFlags::empty();
 
     debug!("Unmounting block device {} (flags={:?}) ...", target, flags);
```

Comment:
There's no real point in having the flags value in the debug message now either

Reply:
Yeah, I kept it just in case it changes in the future but can remove